initgeneration/init.sh.tmpl

```diff
@@ -116,6 +116,25 @@ tenant ${host_tenant}
 isCloudNativeFullStack true" >> ${1}
 }
 
+writeConfEnrichmentTo() {
+  mkdir -p "${1}"
+  echo "k8s.pod.uid ${K8S_PODUID}
+k8s.pod.name ${K8S_PODNAME}
+k8s.namespace.name ${K8S_NAMESPACE}
+dt.kubernetes.workload.kind ${DT_WORKLOAD_KIND}
+dt.kubernetes.workload.name ${DT_WORKLOAD_NAME}
+dt.kubernetes.cluster.id ${cluster_id}" >> "${1}/${2}.properties"
+
+  echo "{
```

Comment:
Just curious here, why do you need the same data in a .properties file/format as well as .json?

Reply:
That's external requirement, see: (OneAgent file-open interface)[https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/#oneagent-file-open-interface]

Reply:
the link says it's fine to have either.
is the documentation wrong or is one format enough? :smile:

Reply:
I would say that the docs are OK. If you go for .json the indirection file will point you to a .json file if you go for .properties you will get redirected to a .properties file. We support both formats as we want to give some freedom to customers how they would like to use it.

Reply:
but is there any reason we would like to provide this freedom for the user? 
I already see the confused customer questions asking what is the difference between the 2 approaches or even weirder questions. (I have experience with people confused by multiple options to do the same thing)

IMO we shouldn't provide multiple configuration options for doing the same thing, without any concrete usecase which tells us the user is only able to use one of the possible ways configure and not the other(s). 
(fewer ways to configure ==> simpler docs + simple implementation ==> less confused users + less buggy code)

Reply:
since it's an external requirement, it's fine to have it duplicated